cpp/src/io/parquet/page_enc.cu

```diff
@@ -1408,33 +1408,33 @@ __device__ bool increment_utf8_at(uint8_t* ptr)
 }
 
 /**
- * @brief Truncate a UTF-8 string to at most truncate_length bytes.
+ * @brief Truncate a span of UTF-8 characters to at most truncate_length bytes.
```

Comment:
Sorry to nitpick further, but this isn't actually true is it? It may not be able to truncate and so "at most truncate_length bytes" isn't correct is it?

```suggestion
 * @brief Attempt to truncate a span of UTF-8 characters to at most truncate_length_bytes.
```

And then below you mention that it may not truncate, so I think we're good there.

Reply:
nice catch!